meilisearch-http/src/option.rs

```diff
@@ -146,6 +146,10 @@ pub struct Opt {
     #[clap(long, env = "MEILI_LOG_LEVEL", default_value = "info")]
     pub log_level: String,
 
+    // Enables Prometheus metrics and /metrics route.
+    #[clap(long, requires = "enable-metrics-route")]
```

Comment:
```suggestion
    /// Enables Prometheus metrics and /metrics route.
    #[clap(long)]
```

Reply:
Hey, I did rebase the main to the feature branch. Still,
![Screenshot from 2022-08-17 21-09-57](https://user-images.githubusercontent.com/43550760/185182977-29d24315-3c59-4200-a3c3-5c1fa03c7dbc.png)
 the Clippy is failing.

Reply:
It's super strange; I can't reproduce locally.
But if you can go on, just fix it here and make it stop.

If you're not familiar with Clippy;
You can run `cargo clippy --fix` to automatically fix most of the warnings.

Then don't forget to run `cargo fmt` and you should be good!